cmd/lncli/cmd_invoice.go

```diff
@@ -67,7 +67,15 @@ var addInvoiceCommand = cli.Command{
 				"payer in reaching you. If amt and amt_msat " +
 				"are zero, a large number of hints with " +
 				"these channels can be included, which " +
-				"might not be desirable.",
+				"might not be desirable. This can be limited" +
+				"with the hints flag.",
+		},
+		cli.Int64Flag{
+			Name: "hints",
```

Comment:
Generally we just give the same name for a param that's used in cli/rpc.

Reply:
`Hints` seems too similar to `RouteHints` but I also wanted the flag name to be short like the other flags.